Symfony/CS/Fixer/Contrib/MethodArgumentDefaultValueFixer.php

```diff
@@ -0,0 +1,203 @@
+                break;
+            }
+
+            if (
+                $this->isDefaultArgumentAfterIndex($tokens, $nextRelevantIndex - 1) &&
+                $this->hasNonDefaultArgumentAfterIndex($tokens, $nextRelevantIndex - 1)
+            ) {
+                $this->removeDefaultArgument($tokens, $nextRelevantIndex);
+            }
+            $examinedIndex = $nextRelevantIndex;
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return bool
+     */
+    private function hasNonDefaultArgumentAfterIndex(Tokens $tokens, $index)
+    {
+        $nextRelevantTokenIndex = $this->findNextVariableOrTokenOfKind($tokens, $index, $this->functionDefinitionTerminatorTokens);
+
+        if (null === $nextRelevantTokenIndex) {
+            return false;
+        }
+
+        while (!in_array($tokens[$nextRelevantTokenIndex]->getContent(), $this->functionDefinitionTerminatorTokens, true)) {
+            $nextRelevantTokenContent = $tokens[$tokens->getNextMeaningfulToken($nextRelevantTokenIndex)]->getContent();
+
+            if (in_array($nextRelevantTokenContent, $this->argumentTerminatorTokens, true)) {
+                return true;
+            }
+
+            $nextRelevantTokenIndex = $this->findNextVariableOrTokenOfKind($tokens, $nextRelevantTokenIndex, $this->functionDefinitionTerminatorTokens);
+        }
+
+        return false;
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param array  $relevantTokens
+     *
+     * @return int|null
+     */
+    private function findNextVariableOrTokenOfKind(Tokens $tokens, $index, array $relevantTokens)
+    {
+        $variableIndices = array_keys($tokens->findGivenKind(T_VARIABLE));
+
+        $nextVariableIndex = $this->getFirstValueBiggerThan($variableIndices, $index);
+        $nextRelevantTokenIndex = $tokens->getNextTokenOfKind($index, $relevantTokens);
+
+        if (null === $nextVariableIndex) {
+            return $nextRelevantTokenIndex;
+        }
+
+        if (null === $nextRelevantTokenIndex) {
+            return $nextVariableIndex;
+        }
+
+        return ($nextVariableIndex < $nextRelevantTokenIndex) ? $nextVariableIndex : $nextRelevantTokenIndex;
+    }
+
+    /**
+     * @param array $values
+     * @param int   $minimumValue
+     *
+     * @return int|null
+     */
+    private function getFirstValueBiggerThan(array $values, $minimumValue)
+    {
+        foreach ($values as $value) {
+            if ($value > $minimumValue) {
+                return $value;
+            }
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return bool
+     */
+    private function hasDefaultValueAfterIndex(Tokens $tokens, $index)
+    {
+        $nextTokenIndex = $tokens->getNextTokenOfKind($index, $this->defaultValueTokens);
+        $nextToken = $tokens[$nextTokenIndex];
+
+        return $nextToken->getContent() === '=';
```

Comment:
Well, it won't matter because I'll use equals anyway there. There won't be a comparison.

Is there a shortcut to check if the token is one of an array? Equals won't right cut it here.